src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/NonEmptyAtclauseDescriptionCheckTest.java

```diff
@@ -95,4 +101,30 @@ public void testCheckTwo() throws Exception {
         };
         verifyWithInlineConfigParser(getPath("InputNonEmptyAtclauseDescriptionTwo.java"), expected);
     }
+
+    /**
+     * This tests that the check does not fail when the input file contains unmappable characters.
+     * The test file contains the character ü which is not mappable to US-ASCII. It makes sure that
+     * unmappable characters are replaced with the default replacement character using
+     * {@link CodingErrorAction#REPLACE}.
+     *
+     * @throws Exception exception
+     */
+    @SuppressForbidden
```

Comment:
What is suppress forbidden for?

Reply:
Verify method, as we supposed to use verify that use input based config. But we need to set charset, and this type set we do not support yet.
So for now old style testing on Input file, without embedded config.